### ISSUE TITLE
[Proposal/concerns] Commission for the withdrawal of vegetables from the farm by level

### ISSUE THREAD

**Issue by IvanSkazkin** [sunflower-land/sunflower-land#575]
**Hello**. I propose to make a commission for the transfer of vegetables. For example. If the account is lvl 1, then the commission is 90% percent for all resources (resources will be burned or tokens will be redistributed for them in the wishing well). If lvl 2 then 70%
If lvl 5 then 50%
If 7 and above 10 percent. If lvl 10 then 0%.
It is the general level of the farm, not the profession.
I think this is how you can defeat a multi-account.
The idea is that it will be difficult for illegal players to start transferring many things to one player at once. Otherwise, it may break the market. It is important that the commission is 0%, the player needs about 1 month of active play.
Then, if the player is caught and banned, he will need another month to farm. 
And it also encourages players to be in the project
**Thanks**
Percentages and levels are subject to change, this is just a concept proposal.

**Comment by macon9923** [comment]
legendary idea

**Comment by IvanSkazkin** [comment]
> легендарное предложение

**Thank you.** I also think this offer will be made more valuable)

**Comment by macon9923** [comment]
Definitely to limit people who sell cheap........I hope the team will take your proposal

**Comment by zTMXz** [comment]
rly nice suggestion. It will keep econimics in good condition

**Comment by DonatasDee** [comment]
What you DEFINITELY should not do, is spam every single language channel begging for attention.

If the proposal has feet, it will get attention that it deserves - it's not a popularity contest.

**Comment by IvanSkazkin** [comment]
> What you DEFINITELY should not do, is spam every single language channel begging for attention.
> 
> If the proposal has feet, it will get attention that it deserves - it's not a popularity contest.

I didn't mean to send spam. I just wrote my suggestion for each language user. Therefore, it cannot be spam.

**Comment by Kubeki** [comment]
I agree with him there is no real mechanic to ensure such bad actors in game, we have to protect the community by 

implementing some severe actions otherwise i can't see how we can counter it i've saw and linked to Adam two exploiters on 

opensea they had literally crafted 7 carrot sword, it means that people still exploit the system.

**Comment by Nazarey25** [comment]
Поддерживаю! идёя толковая,  месяц люди не будут  развивать фермы  и мультиачить )

**Comment by Nurdian185** [comment]
I'm totally agree with this

**Comment by thisUsernameAlreadyExists** [comment]
I think withdrawing should not be available if your farmer is not at least lv 5.

**Comment by adamhannigan** [comment]
@IvanSkazkin Great idea 👏 

We definitely will explore taxes on withdrawing resources from your farm (especially if it is a brand new farm) to help improve the game economy.

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
